app/templates/index.hbs

```diff
@@ -3,7 +3,7 @@
     <div class="au-c-sidebar">
       <div class="au-c-sidebar__content">
         <div class="au-c-sidebar__header">
-          <p class="au-u-h5 au-u-medium">Menu modules</p>
+          <h6 class="au-u-h6 au-u-medium">Menu modules</h6>
```

Comment:
I don't think it needs to be a h6 because it's styled like one?

Reply:
Can be another h* but as it's a title it should be marked up as a title element. A p tag does not seem correct here.

Reply:
Yea, fair enough. I usually don't use h elements if the thing shouldn't be visible in the fictive "outline" which is why we used a p tag here.

I think using a h6 is a weird choice as well :smile: since it just seems completely dependent on the styling. I'm no expert on semantics and the spec isn't very clear either so I'll drop it.